collectors/cronjob.go

```diff
@@ -150,6 +156,7 @@ func (jc *cronJobCollector) collectCronJob(ch chan<- prometheus.Metric, j v2batc
 	}
 
 	addGauge(descCronJobInfo, 1, j.Spec.Schedule, string(j.Spec.ConcurrencyPolicy))
+	addGauge(descCronJobCreated, float64(j.CreationTimestamp.Unix()))
```

Comment:
The `CreationTimestamp` [is not necessarily set](https://github.com/julia-stripe/kube-state-metrics/blob/02ec0772506b4d95ca06de7f3fa3d4587785be9a/vendor/k8s.io/client-go/pkg/api/v1/types.go#L141-L150). So we should skip this metric whenever `CreationTimestamp == nil`.